### PR TITLE
MM 1826 Add support for channel titles with right to left languages

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

File: web/react/components/sidebar.jsx [hunk 1]
```diff
@@ -473,4 +478,10 @@ module.exports = React.createClass({
             </div>
         );
     }
-});
+}
+
+Sidebar.displayName = 'Sidebar';
```
Comment (crspeller) [inline web/react/components/sidebar.jsx]: This is unnecessary. The display name is now the class name.

**Comment by nickago** [comment]
Ready for re-review

File: web/react/utils/utils.jsx [hunk 2]
```diff
@@ -753,8 +753,23 @@ module.exports.isValidUsername = function(name) {
     return error;
 };
 
+function checkRTL(string) {
```
Comment (coreyhulen) [inline web/react/utils/utils.jsx]: We should think about using a library here.  I'm shocked there isn't built in support into JS for this.

Comment (nickago) [inline web/react/utils/utils.jsx]: I'm open to using a library, but it seems like a simple enough and stable enough thing to just use a regex of what is what type.

Comment (coreyhulen) [inline web/react/utils/utils.jsx]: Lets chat.  I think there is a problem with the original bug description.  We should use the browsers encoding to detect RHL and display appropriately.

**Pull request closed without merging**